spec/index.bs

```diff
@@ -148,11 +171,15 @@ spec: WebIDL; urlPrefix: https://heycam.github.io/webidl/#
   The sync event is considered to run <dfn>in the background</dfn> if no <a>service worker clients</a> whose <a>frame type</a> is top-level or auxiliary exist for the origin of the corresponding service worker registration.
 
   The user agent is considered to be <dfn>online</dfn> if the user agent has established a network connection. A user agent MAY use a stricter definition of being <a>online</a>. Such a stricter definition MAY take into account the particular <a>service worker</a> or origin a <a>sync registration</a> is associated with.
+
+  A <dfn>one-shot Background Sync opportunity</dfn> allows one-off synchronization between the server and the web app, and can be requested via a <a>sync registration</a>.
+  
+  A <dfn>periodic Background Sync opportunity</dfn> allows periodic synchronization between the server and the web app, the exact frequency of which is decided by the user agent. It can be requested via a <a>periodic sync registration</a>.
 </section>
 
 <section>
   <h2 id="constructs">Constructs</h2>
-  A <a>service worker registration</a> has an associated <dfn>list of sync registrations</dfn> whose element type is a <a>sync registration</a>.
+  A <a>service worker registration</a> has an associated <dfn>list of sync registrations</dfn> whose element type is a <a>sync registration</a>. It also has an associated <dfn>list of periodicSync registrations</dfn> whose element type is a <a>periodic sync registration</a>.
 
   A <dfn>sync registration</dfn> is a tuple consisting of a <a>tag</a> and a <a lt="registration state">state</a>.
```

Comment:
I still think the definitions of the enum values should be moved here, and referenced below